src/main/java/org/example/Calculator.java

```diff
@@ -0,0 +1,59 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class Calculator {
+    public static void main(String[] args) {
```

Comment:
If it's not an entry point of the program it should be renamed to some verb like main -> processInput(). Also args is never used